project.clj

```diff
@@ -76,7 +76,7 @@
 
   :plugins [[lein-release "1.0.5" :exclusions [org.clojure/clojure]]]
 
-  :uberjar-name "puppet-server-release.jar"
+  :uberjar-name "puppetserver-release.jar"
```

Comment:
This should be fine. The ezbake init scripts use :uberjar-name directly. I *think* we may have instructions for using HUP that would need updating though.